src/WebJobs.Script/Utility.cs

```diff
@@ -451,6 +451,17 @@ public static string GetValueFromState<TState>(TState state, string key)
             }
             return value;
         }
+
+        public static string GetValueFromScope(IDictionary<string, object> scopeProperties, string key)
+        {
+            object value;
+            if (scopeProperties != null && scopeProperties.TryGetValue(key, out value) && value != null)
+            {
+                return value.ToString();
+            }
+            return string.Empty;
```

Comment:
If the value wasn't found, you likely want to return null here. That helps you differentiate between an actual `string.Empty` value in the scope, and one that simply didn't exist.

Then in your usage in the Logger, you'd have to add `?? string.Empty` to make sure you're coalescing the value that you need.